### PR TITLE
Deduplicate Helix environment setup in send-to-helix-step.yml

### PR DESCRIPTION
**Pull request by trylek**

### CODE REVIEW

File: eng/pipelines/common/templates/runtimes/send-to-helix-step.yml [hunk 1]
```diff
@@ -26,60 +26,16 @@ parameters:
...
-      env:
-        # Arcade uses this SDK instead of trying to restore one.
-        DotNetCoreSdkDir: /usr/local/dotnet
-
-  - script: $(Build.SourcesDirectory)/eng/common/msbuild.sh --ci ${{ parameters.helixProjectArguments }} /maxcpucount /bl:$(Build.SourcesDirectory)/artifacts/log/SendToHelix.binlog
-    displayName: ${{ parameters.displayName }}
-    ${{ if ne(parameters.condition, '') }}:
-      condition: ${{ parameters.condition }}
-    env:
+    ${{ if eq(parameters.condition, '') }}:
```
Comment (safern) [inline eng/pipelines/common/templates/runtimes/send-to-helix-step.yml:36]: you could remove this and make the default condition value succeeded().

Comment (trylek) [reply eng/pipelines/common/templates/runtimes/send-to-helix-step.yml:36]: I like the suggestion but I'm not sure if I understand it properly - are you saying that I should put <code>condition: succeeded()</code> in the parameters section? We're not using such construct anywhere in our scripts, is that going to work? Or did you mean just putting <code>condition: true</code> there and using the <code>succeeded()</code> function in construction of the "final" condition like we do in several scripts, something like <code>condition: and(succeeded(), ${{ parameters.condition }})</code>?

Comment (safern) [reply eng/pipelines/common/templates/runtimes/send-to-helix-step.yml:36]: I think we should do: `condition: true` in the parameters section and then use it like: `condition: and(succeeded(), ${{ parameters.condition }})`

Comment (trylek) [reply eng/pipelines/common/templates/runtimes/send-to-helix-step.yml]: Thanks, fixed.

**Review by safern — APPROVED** [review]

**Pull request merged**